apps/workflows/urls.py

```diff
@@ -16,12 +17,14 @@
 urlpatterns = [
     path("", index.as_view()),
     path("healthy", healthy.as_view()),
-    path(f"api/{WORKFLOWS_API_VERSION}", workflows.as_view()),
-    path(f"api/{WORKFLOWS_API_VERSION}/<str:pk>", classification.as_view()),
-    path(f"api/{WORKFLOWS_API_VERSION}/<str:pk>/adjust", adjust.as_view()),
+    path(f"api/{WORKFLOWS_API_VERSION}/workflows", workflows.as_view()),
+    path(f"api/{WORKFLOWS_API_VERSION}/workflows/<str:pk>", classification.as_view()),
+    path(f"api/{WORKFLOWS_API_VERSION}/workflows/<str:pk>/adjust", adjust.as_view()),
     path(f"api/{WORKFLOWS_API_VERSION}/graph/workflows", graph_workflows.as_view()),
     path(
         f"api/{WORKFLOWS_API_VERSION}/graph/workflows/<str:pk>",
         graph_classification.as_view(),
     ),
 ]
+
+urlpatterns += deprecated_urlpatterns
```

Comment:
I wonder whether there is any value in preserving the deprecated paths. The whole workflows (former osim) API is rather an experimental thing and whoever would integrate with it already is probably doomed anyway. @costaconrado WDYT?

Reply:
I know that anything red in openapi.yml is normally allowed in the major versions only but we already agreed before (with **taskman**) that we will exclude the experimental stuff from the guarantees.

Reply:
I agree. The workflows we had before the changes did not reflect any real process we had in place, as you mentions it was experimental and I don't think anyone had integrated+used it before (probably only griffon through automation which would not be a problem), I believe it is safe to just remove this without deprecation.

Reply:
I preserved them mainly because it was a breaking change. But if they are safe to remove, I'll happily remove them.

Reply:
So I say these endpoints I experimental and safe to be removed so the responsibility is mine :smile: and you can proceed with the removal. It will simplify the code and I really see no drawbacks.